Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:11
```

Comment:
Maybe use `FROM node:10-alpine`:
* Long term support
* We can share the docker base image with `Nitro-Backend` and `Nitro-Web`

Reply:
Unfortunately the alpine version of node doesn't work with the dependencies we have. We use the package "sharp", which needs several build tools that node-alpine doesn't include. Manually installing those requirements in the Dockerfile for node-alpine unfortunately didn't work out, so I was forced to use the full node image.
Should we downgrade to version 10 of node though?

Reply:
ewwww

OK, let's just keep it for now